Makefile

```diff
@@ -5,6 +5,8 @@ PLUGN_URL ?= https://github.com/dokku/plugn/releases/download/v0.2.2/plugn_0.2.2
 SIGIL_URL ?= https://github.com/gliderlabs/sigil/releases/download/v0.4.0/sigil_0.4.0_Linux_x86_64.tgz
 STACK_URL ?= https://github.com/gliderlabs/herokuish.git
 PREBUILT_STACK_URL ?= gliderlabs/herokuish:latest
+DOKKU_USER ?= dokku
+DOKKU_ROOT ?= /home/${DOKKU_USER}
```

Comment:
Edge case nitpick: What if `/home` doesn't exist? For example, what if I use `/usr/homes` for my home dir root or something like that?